plugins/net_plugin/net_plugin.cpp

```diff
@@ -1198,6 +1198,10 @@ namespace eosio {
 
    // called from connection strand
    void connection::_close( connection* self, bool reconnect, bool shutdown ) {
+      if (!self->socket_open){
+         peer_ilog(self, "attempt to close connection while it is already closed. skipping");
```

Comment:
Did you mean to add a `return` here?
Also I think this should be a `peer_dlog`.

Reply:
yep, silly error